docs/src/notebooks/5_basic_stdmodel_construction.jl

```diff
@@ -55,28 +55,14 @@ m3 = metabolite_list[3]
 m4 = metabolite_list[4]
 
 @add_reactions! model begin # macro approach
-    "r4", m2 ⟶ m4, 0, 1000
-    "r_m3", m3 ⟷ nothing, -1000, 1000
-    "r_m4", m4 ⟶ nothing
-    "r5", m4 ⟶ m2
+    "r4", m2 → m4, 0, 1000
+    "r_m3", m3 ↔ nothing, -1000, 1000
+    "r_m4", m4 → nothing
+    "r5", m4 → m2
 end
 
 model.reactions["r4"].grr = [["g5"], ["g6", "g7"], ["g8"]]
 
-#md # !!! note "Note: Using reaction arrows"
-#md #       `COBREXA` exports arrows that can be used to construct reactions.
-#md #       Both the long and short arrows (`⟶ == →`) mean the same thing if they
-#md #       point in the same direction.
-#md #
-#md #       These arrows are accessible by using the `LaTeX` completions built into
-#md #       Julia. For example:
```

Comment:
maybe it is good to keep a note on how to enter these arrows 😄

Reply:
I actually think that the long version of these arrows provides an increased readability, not sure what you are thinking of it.

Reply:
Yep, I also like the long versions more, they display better. Hmm now that I have ligatures, it would be cool if we could replace the arrows with <-->, <--, and -->

Reply:
Is it even possible (easy) to do that? How does julia interpret things like `-->`, can we define what that means somehow? It would be really cool

Reply:
custom operators are not supported by julia

\bigfrown

Reply:
(I added the note back)